src/RecoveryServices/RecoveryServices.Backup.ServiceClientAdapter/RecoveryServices.Backup.ServiceClientAdapter.csproj

```diff
@@ -14,6 +14,7 @@
   <ItemGroup>
 	  <PackageReference Include="Microsoft.Azure.Management.RecoveryServices" Version="4.3.3-preview" />
 	  <PackageReference Include="Microsoft.Azure.Management.RecoveryServices.Backup" Version="5.0.0-preview" />
+	  <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="3.17.4-preview" />
```

Comment:
The static issue basically means when you introduce a new reference, you need to add the assembly of the library to the "RequiredAssemblies" filed of your module's psd1.
However we have generated some common clients so the SDK isn't necessary.
Can you use this [FeatureClient](https://github.com/Azure/azure-powershell-common/blob/main/src/ResourceManager/Version2016_09_01/Generated/FeatureClient.cs#L36) instead?

Reply:
sure, let me check. Thanks!